Update library to reflect teleport install systemd addition (#15602)
gravitational/teleport: docs/pages/getting-started/linux-server.mdx

  </TabItem>

  <TabItem label="Source or custom install">

>   1. Create a systemd unit file using the `teleport install systemd` command and the [unit file path](https://www.freedesktop.org/software/systemd/man/systemd.unit.html) you'd

Reviewer: I recommend moving additional articles and links to the last section "Next steps" to avoid [breaking readers focus](https://goteleport.com/docs/contributing/documentation/style-guide/#expectations) according to our style guide.

Reviewer 2: Moved all links to the "Next Steps" section, retitled from "Further Reading" to better fit our style guide

>   like to use for your configuration. The `/etc/systemd/system/teleport.service` path is recommended in most use case:

Reviewer: ```suggestion
  like to use for your configuration. We recommend `/etc/systemd/system/teleport.service` path in most use cases:
```

Reviewer: style nit: fixed the typo and switched to active voice.

Reviewer 2: Switched to the active voice here and in more places, fixed typo

    ```code
    $ sudo teleport install systemd -o /etc/systemd/system/teleport.service
    ```
  2. Enable and start the new teleport service daemon:

    ```code
    $ sudo systemctl enable teleport && sudo systemctl start teleport
    ```
  </TabItem>
</Tabs>

<Admonition title="The Teleport daemon" type="tip">
Configuration for the teleport daemon relies on [systemd](https://www.freedesktop.org/wiki/Software/systemd/). For more information on how the

> teleport service daemon is configured, see our guide on how to [Run Teleport as a Daemon](https://goteleport.com/docs/setup/admin/daemon/).

Reviewer: ```suggestion
teleport service daemon is configured, see our guide on how to [Run Teleport as a Daemon](../setup/admin/daemon.mdx).
```

Reviewer: Using internal links is better when possible, we will verify them more efficiently.

Reviewer 2: Good point, I completed this for all `goteleport.com` links
